main.py

```diff
@@ -183,6 +183,11 @@ def notify_uplink_detected(self, event):
                                             'interface_b': interface_b})
             self.controller.buffers.app.put(event_out)
 
+    def notify_lldp_change(self):
+        """Dispatch a KytosEvents to notify changes to the LLDP status."""
+        event_out = KytosEvent(name='kytos/topology.network_status.updated')
```

Comment:
This event is created by `kytos/lldp`, so its name should be `kytos/lldp....` instead of `kytos/topology....`.

Reply:
Resolved in comment 608c17b.